ProjectName/src/navigation/NavTabs.tsx

```diff
@@ -94,6 +94,12 @@ const Tabs = () => {
               color={focused ? 'purple' : 'gray'}
             />
           ),
+          headerStyle: {
+            backgroundColor: darkmode == 'dark' ? '#212529' : 'transparent',
```

Comment:
I would suggest extract color variables to atleast some separate file. This will help to create a single point of responsibility. And also would help to remove duplication.